Nathan

nable941 opened the pull request:
Update to the player file with a character creation method. Includes notation on integrating it into the GUI 2 button system

nable941 committed:
Merge pull request #9 from Crstt/main

update to main

nable941 committed:
Merge pull request #16 from Crstt/main

update

nable941 committed:
Update player class to text

nable941 committed:
Merge pull request #25 from Crstt/main

updating sync

nable941 committed:
Merge pull request #27 from Crstt/main

update local

nable941 committed:
Updating player for creation

creating player creation method

nable941 committed:
Merge branch 'Nathan' of https://github.com/Crstt/SDEV265_GroupRed into Nathan

nable941 committed:
Player Update

Updated player with character creation function. Function includes notes on connection points to make it operable with the GUI. See also the test file to see none GUI functionality

Crstt commented on player.py:
```diff
@@ -1,28 +1,56 @@
         self.food = food
         self.huntAdjust = huntAdjust
-        self.buyAdjust = buyHudjust
+        self.buyAdjust = buyAudjust
```
@christopherhochgesang You should use this variable in the river scenario instead of using a static adjustment

nable941 replied:
It's not a static adjustment. It is character based depending on the selected character just as the huntAdjust is, and therefore an attribute of the player not the scenario.

Crstt replied:
Yes but Chris has a line that didn't use this. So he should. Sorry this comment was for him
```python
# Pay for ferry choice Choice
                    elif(playerChoice==2):
                        if(self.player.character=="Merchant"):
                            ferryCost = self.ferryCost*.9
```

Crstt commented on player.py:
```diff
@@ -1,28 +1,56 @@
-                #checkpointManager
-                pass
+def selectCharacter():
+    filename = "characters.txt"
```
If it is a csv it should have the CSV file type

nable941 replied:
Updated

Crstt commented on characters.txt:
Should use the CSV file type

nable941 replied:
changed

Crstt commented on player.py:
```diff
@@ -1,28 +1,56 @@
+        #create character description for display
+        hunt = str(float(choices[cKey[i]].huntAdjust) * 100)
+        buy = str(float(choices[cKey[i]].buyAdjust) * 100)
+        discriptionLabel = "The" + choices[cKey[i]].character + " starts with $" + choices[cKey[i]].money + ", and a " + hunt + " percent change to hunting as well as " + buy  +" percent change to buying costs."
```
Typo "description" not "discription"

nable941 replied:
Corrected

Crstt reviewed (changes requested):
Great job! Just a few very small tweaks.

nable941 committed:
Change Request update

Fixed Type-o in variable name and updated characters.txt to characters.csv and corrected file open to reflect change.
